integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciAvailabilityDefault.java

```diff
@@ -71,4 +74,15 @@ static boolean canReach(String address,
         }
     }
 
+    private static String getOpcPath() {
+        String input = METADATA_SERVICE_BASE_URL;
+        int index = -1;
+        for (int nth = 3; nth > 0; nth--) {
+            index = input.indexOf("/", index + 1);
+            if (index == -1) {
```

Comment:
Is this really what we want to do here, return null? Seems like we should be failing fast with an exception.  And since we want to catch this when we bump OCI versions, we should write a test for this method in the PR too.  That way we catch it in our build environment and not in the users.

Reply:
Updated based on comments, i.e. getOpcPath() now throwing exception during failure and also added unit test coverage